homework/index.js

```diff
@@ -1,46 +1,169 @@
+      createAndAppend('p', infoRight, {
+        class: 'no-content',
+        text: 'There are no contributions yet.',
+      });
+      return;
+    }
+
+    contributors.forEach(contributor => {
+      const contributorLink = createAndAppend('a', infoRight, {
+        class: 'contributor-link',
+        href: contributor.html_url,
+        target: '_blank',
+      });
+      const imgDiv = createAndAppend('div', contributorLink, { class: 'contributor-img-div' });
+      createAndAppend('img', imgDiv, {
+        class: 'contributor-img',
+        src: contributor.avatar_url,
+        alt: 'Contributor photo',
+      });
+      imgDiv.firstChild.style.maxHeight = '50px';
+      const dataDiv = createAndAppend('div', contributorLink, { class: 'contributor-data-div' });
+      createAndAppend('div', dataDiv, {
+        class: 'contributor-name',
+        text: contributor.login,
+      });
+      createAndAppend('div', dataDiv, {
+        class: 'contributor-contributions',
+        text: contributor.contributions,
+      });
+      createAndAppend('hr', infoRight, {
+        class: 'hr-contributor',
+      });
+    });
+  }
+
+  async function bringContributions(urlText, container) {
+    const infoRight = createAndAppend('div', container, {
+      class: 'info-right',
+      text: 'Contributions:',
     });
+    try {
+      const contributors = await bringData(urlText);
+      renderContributions(contributors, infoRight);
+    } catch (error) {
+      renderError(error, infoRight);
+    }
   }
 
+  function showRepoInfo(wantedRepo, container) {
+    const infoLeft = createAndAppend('div', container, {
+      class: 'info-left',
+      text: 'Repository information:',
+    });
+    const table = createAndAppend('table', infoLeft);
+    const tbody = createAndAppend('tbody', table);
+    const firstRow = addRow(tbody, `Name `);
+    const secondRow = addRow(tbody, `Description `, wantedRepo.description);
+    addRow(tbody, `Forks `, wantedRepo.forks);
+    addRow(tbody, `Last update `, new Date(wantedRepo.updated_at).toLocaleString());
+    createAndAppend('a', firstRow.lastChild, {
+      text: wantedRepo.name,
+      href: wantedRepo.html_url,
+      target: '_blank',
+    });
+    if (wantedRepo.description === null) {
+      createAndAppend('span', secondRow.lastChild, {
+        class: 'no-content',
+        text: 'There is NO description.',
+      });
+    }
+  }
+
+  const deleteContents = container => {
+    while (container.firstChild) {
+      container.removeChild(container.firstChild);
+    }
+  };
+
+  function renderSelectElement(repositories, selectEl, infoContainer) {
+    repositories.sort((a, b) => a.name.localeCompare(b.name));
+    repositories.forEach((repo, index) => {
+      createAndAppend('option', selectEl, { text: repo.name, value: index });
+    });
+    showRepoInfo(repositories[selectEl.value], infoContainer);
+    bringContributions(repositories[selectEl.value].contributors_url, infoContainer);
```

Comment:
It is fine to use `selectEl.value` here, but note this value with initially be zero (the value attribute of the first `<option>` element).

Reply:
yes. when the page is loaded this the value is sure zero. so it is better to write zero directly is this case. 
thanks for the review and best regards.